homeassistant/components/binary_sensor/zwave.py

```diff
@@ -0,0 +1,70 @@
+"""
+homeassistant.components.binary_sensor.zwave
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Interfaces with Z-Wave sensors.
+
+For more details about this platform, please refer to the documentation
+at https://home-assistant.io/components/zwave/
+"""
+
+
+import logging
+
+from homeassistant.components.zwave import (
+    ATTR_NODE_ID, ATTR_VALUE_ID,
+    COMMAND_CLASS_SENSOR_BINARY, NETWORK,
+    ZWaveDeviceEntity)
+from homeassistant.components.binary_sensor import (
+    DOMAIN,
+    BinarySensorDevice)
+
+_LOGGER = logging.getLogger(__name__)
+DEPENDENCIES = []
+
+
+def setup_platform(hass, config, add_devices, discovery_info=None):
+    """Setup the mysensors platform for sensors."""
+
+    if discovery_info is None or NETWORK is None:
+        return
+
+    node = NETWORK.nodes[discovery_info[ATTR_NODE_ID]]
+    value = node.values[discovery_info[ATTR_VALUE_ID]]
+
+    value.set_change_verified(False)
+    if value.command_class == COMMAND_CLASS_SENSOR_BINARY:
+        add_devices([ZWaveBinarySensor(value, "opening")])
```

Comment:
All z-wave binary sensors should not be of the "opening" type. It is just used because I was lazy :)

openzwave defines an [enumeration](https://github.com/openhab/openhab/blob/master/bundles/binding/org.openhab.binding.zwave/src/main/java/org/openhab/binding/zwave/internal/protocol/commandclass/ZWaveBinarySensorCommandClass.java#L184) for binary sensor types. Assuming the python bindings for openzwave deliver that, we should add automatic mapping between those and the homeassistant binary_sensor SENSOR_CLASSES.

I can look at adding this during the next couple of days.